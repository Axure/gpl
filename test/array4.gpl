void main()
{
	int[2] arr_i;
	string[3] arr_s;
	arr_i[1] = 9;
	print(arr_i[1]);
	arr_i[1] = 7;
	print(arr_i[1]);
	arr_s[0] = "success";
	print(arr_s[0]);
	arr_s[0] = "dog";
	print(arr_s[0]);
}