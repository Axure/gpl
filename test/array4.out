9
7
success
dog
